web/source/kmwbase.ts

```diff
@@ -303,12 +303,17 @@ namespace com.keyman {
     }
 
     /**
-     *  Add default or all keyboards for a given language
+     *  Add default keyboards for given language(s)
      *
-     *  @param  {string}   arg    Language name (multiple arguments allowed)
+     *  @param  {string|string[]}   arg    Language name (multiple arguments allowed)
+     *  @returns {Promise<KeyboardStub[]>} Promise of added keyboard stubs
      **/
-    ['addKeyboardsForLanguage'](arg) {
-      this.keyboardManager.addLanguageKeyboards(arguments);
+    ['addKeyboardsForLanguage'](arg: string[]|string) : Promise<com.keyman.keyboards.KeyboardStub[]> {
+      if (typeof arg === 'string') {
+        return this.keyboardManager.addLanguageKeyboards(arg.split(',').map(item => item.trim()));
+      } else {
+        return this.keyboardManager.addLanguageKeyboards(arg);
```

Comment:
_**If**_ we wanted to preserve the 'alert' stuff we've already been doing within KMW, this is a good spot to add a `.catch()` clause to intercept errors and report their messages.  That said, the uncaught rejected `Promise`s should leave `console.error` messages.  Also, `util.alert` is [still part of our published API](https://help.keyman.com/DEVELOPER/engine/web/14.0/reference/util/alert), so we could simply direct site-devs to call it from their own `.catch()` blocks; it's a change from before, but is very minor and an easy-to-implement change.

If we do favor handling it automatically (rather than deferring to site devs), please re-throw the errors so that site developers can also add on their own `.catch()` clauses.

Reply:
Clarifying with @jahorton 
> It's likely better to just leave alert-generation to the site's maintainer

I'll leave this part intact, and check the option flag for alerts on a future PR